Added list of commands with short descriptions to USAGE.md

anfly0 opened the pull request:
Not 100% verified, but close to it.
Should at least serve as a good start.

shnupta commented on USAGE.md:
```diff
@@ -3,4 +3,118 @@
+>	- __ctrl + n__ - Jump back to previous position after __ctrl + m__ .
+
+2. ### Edit
+>	- __rc__ - Copy current row.
```
The following should be reversed. e.g. `cr` instead of `rc`

shnupta commented on USAGE.md:
```diff
@@ -3,4 +3,118 @@
+2. ### Misc
+>	- __q!__ - Force quit without saving changes.
+>	- __w__ - Save changes.
+>	- __wp__ - Save changes and quit.
```
typo `wp` should be `wq`

shnupta commented on USAGE.md:
```diff
@@ -3,4 +3,118 @@
+>	- __pc__ - Paste from clipboard.
+
+3. ### Misc
+>	- __cc__ - ???.
```
I don't believe there is a `cc` command at the moment

shnupta commented:
Hey, nice start. We can keep adding to this.

We should also probably add links to each section at the beginning of the file.

anfly0 commented:
I've fixed the typos and and added some rudimentary anchor links.
But i'm still wondering about the cc command in Normal mode, this is a part of the "Normal mode" switch statement: 
https://github.com/shnupta/bric/blob/2ed07f1a3f679298e742a0f4a553e7982b75f572/bric.c#L2026-L2028

anfly0 commented:
I've fixed the typos and added some rudimentary anchor links.
But i'm still wondering about the 'cc' command in Normal mode, this is a part of the "Normal mode" switch statement : 
https://github.com/shnupta/bric/blob/2ed07f1a3f679298e742a0f4a553e7982b75f572/bric.c#L2026-L2028

shnupta commented:
Ohhh yes sorry my bad. That's to clear whatever is in the clipboard.
